includes/compat/class-wc-stripe-email-admin-failed-refund.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * An email sent to the admin when a refund fails.
+ *
+ * @since 9.6.0
+ */
+class WC_Stripe_Email_Admin_Failed_Refund extends WC_Stripe_Email_Failed_Refund {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->id          = 'failed_refund_admin';
```

Comment:
Should we use a more specific string to avoid possible collisions with other gateways?
```suggestion
		$this->id          = 'wc_stripe_failed_refund_admin';
```

Reply:
I think it makes sense. I used this format to follow the existing standard (all the other emails lack the `wc_stripe` prefix). I just wanted to note in case we want to update them in the future. I have applied your suggestion.

Reply:
Backwards compatibility is tough. 😐 

I vote we don't touch older emails, but we can probably align on plugin-specific IDs moving forward.